install.ps1

```diff
@@ -10,6 +10,19 @@ param(
     [switch] $PreRelease
 )
 
+#Specify target user if different from account running the script, else comment this out
+# $targetUser = "example.username"  # Optional - specify user. Otherwise, uncomment the below
+$targetUser = $Env:UserName
+
+$targetUserObject = New-Object System.Security.Principal.NTAccount($targetUser)
+$targetUserSID = $targetUserObject.Translate([System.Security.Principal.SecurityIdentifier]).value
+
+echo "Changing settings for user $targetUser with SID $targetUserSID"
+
+$Env:LocalAppData = "C:\Users\$targetUser\AppData\Local"
```

Comment:
Any reason on why `C:` is hard coded here? Not everyone installs Windows on that drive.

Reply:
No that's a great point. I'll have a look for a better way, do you have any suggestions before I look into it?

Reply:
You might try to see if `systemroot` environment variable can be used.